kview/src/main/java/dev/ikm/komet/kview/events/EditConceptFullyQualifiedNameEvent.java

```diff
@@ -17,38 +17,59 @@
 
 import dev.ikm.komet.framework.events.Evt;
 import dev.ikm.komet.framework.events.EvtType;
+import dev.ikm.komet.kview.mvvm.model.DescrName;
 import dev.ikm.tinkar.common.id.PublicId;
 
 public class EditConceptFullyQualifiedNameEvent extends Evt  {
 
     public static final EvtType<EditConceptFullyQualifiedNameEvent> EDIT_FQN = new EvtType<>(Evt.ANY, "EDIT_FQN");
 
-    private PublicId publicId;
+    private final PublicId publicId;
 
+    private final DescrName descrName;
 
-    public EditConceptFullyQualifiedNameEvent(Object source, EvtType eventType) {
-        super(source, eventType);
-    }
+//    private final ViewProperties viewProperties;
     /**
      * Constructs a prototypical Event.
      *
      * @param source    the object on which the Event initially occurred
-     * @param eventType
-     * @param publicId
+     * @param eventType the edit event type
+//     * @param viewProperties the view properties for UI.
```

Comment:
do we want to delete this instead of comment it out?

Reply:
Sorry missed deleting it.